java/libphonenumber/test/com/google/i18n/phonenumbers/ExampleNumbersTest.java

```diff
@@ -35,8 +35,7 @@
  */
 public class ExampleNumbersTest extends TestCase {
   private static final Logger LOGGER = Logger.getLogger(ExampleNumbersTest.class.getName());
-  private PhoneNumberUtil phoneNumberUtil =
-      PhoneNumberUtil.createInstance(PhoneNumberUtil.DEFAULT_METADATA_LOADER);
+  private PhoneNumberUtil phoneNumberUtil = PhoneNumberUtil.getInstance();
```

Comment:
I know it's supposed to do the same thing, but I would prefer this stay the way it was, to be sure that this change is not going to affect clients. Tests should still pass. @dlyongemallo thoughts?